manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+    "app-id": "org.example.Ledger",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.38",
+    "sdk": "org.gnome.Sdk",
+    "command": "ledger",
+    "modules": [
+        {
+            "name": "ledger",
+            "buildsystem": "meson",  // Adjust this to your app's build system
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://example.com/ledger-1.0.tar.gz",  // Replace with your app's source URL
+                    "sha256": "abcd1234abcd1234abcd1234abcd1234abcd1234abcd1234abcd1234abcd1234"  // Replace with the actual SHA256 of your app's source archive
+                }
+            ]
+        }
+    ],
+    "finish-args": [
```

Comment:
finish-args is supposed to be above `modules` by convention? Where did you copy that from so that we get this fixed...